primefaces/src/main/java/org/primefaces/component/dataview/DataViewGridItem.java

```diff
@@ -23,6 +23,8 @@
  */
 package org.primefaces.component.dataview;
 
+import jakarta.faces.component.FacesComponent;
+@FacesComponent(value = DataViewGridItem.COMPONENT_TYPE, namespace = DataViewGridItem.COMPONENT_FAMILY)
```

Comment:
Looks like there's some formatting quirks in some of these files that didn't already have any imports.

Reply:
yeah some space look missing!